src/mapping.ts

```diff
@@ -280,16 +286,31 @@ export function handleDelegateVotesChanged(event: DelegateVotesChanged): void {
   daoMetric.save()
 }
 
-export function handleOperatorConfirmed(event: OperatorConfirmed): void {
-  let operator = ConfirmedOperator.load(
-    event.params.stakingProvider.toHexString()
-  )
-  if (!operator) {
-    operator = new ConfirmedOperator(event.params.stakingProvider.toHexString())
+export function handleOperatorBonded(event: OperatorBonded): void {
+  const stakingProvider = event.params.stakingProvider
+  const operator = event.params.operator
+  const timestamp = event.params.startTimestamp
+
+  const preApplication = getOrCreatePreApplication(stakingProvider)
+  if (operator == Address.zero()) {
```

Comment:
It is always safer to use `===` instead of `==`, unless there is a really good reason to use `==`.

Reply:
Done! thank you for the tip